{{cookiecutter.app_slug}}/{{cookiecutter.app_module}}/_version.py

```diff
@@ -43,7 +43,7 @@ def get_config():
     cfg.style = "pep440"
     cfg.tag_prefix = ""
     cfg.parentdir_prefix = "None"
-    cfg.versionfile_source = "audience_utils/_version.py"
```

Comment:
oops looks like that internal top sekret maxpoint code leaked out 😝

Reply:
Oh darn. Interesting why that's even in there considering it started from a fresh project.